tests/e2e/certrotation/certrotation_test.go

```diff
@@ -0,0 +1,112 @@
+package secretsencryption
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+	"testing"
+
+	"github.com/k3s-io/k3s/tests/e2e"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+// Valid nodeOS: generic/ubuntu2004, opensuse/Leap-15.3.x86_64
+var nodeOS = flag.String("nodeOS", "generic/ubuntu2204", "VM operating system")
```

Comment:
here is not better merge var declarations ? 

```
var (
	// Valid nodeOS: generic/ubuntu2004, opensuse/Leap-15.3.x86_64
	nodeOS            = flag.String("nodeOS", "generic/ubuntu2004", "VM operating system")
	agentCount        = flag.Int("agentCount", 1, "number of agent nodes")
	ci                = flag.Bool("ci", false, "running on CI")
)
```

Reply:
This is the standard use in the other vagrant files.